test/blade/RootMintableERC1155Predicate.test.ts

```diff
@@ -1,361 +0,0 @@
-import { expect } from "chai";
```

Comment:
Hopefully we do not miss some of the test cases that were implemented here... :slightly_smiling_face: